pkg/tests/tests.go

```diff
@@ -56,3 +53,27 @@ var tests = []helpers.TestOptions{
 		Handler:  handlers.TestStaticEndpoint,
 	},
 }
+
+func accessReviewTest() helpers.TestOptions {
+	t := helpers.TestOptions{
+		TestName: "access-review",
+		Path:     "/api/authorizations/v1/access_review",
+		Method:   http.MethodPost,
+		Handler:  handlers.TestStaticEndpoint,
+	}
+	resourceReviewReq, err := authv1.NewAccessReviewRequest().
+		AccountUsername("rhn-support-tiwillia").
+		Action("get").
+		ResourceType("Subscription").
+		Build()
+	if err != nil {
+		log.Fatalf("building `access-review` reques: %s", err)
```

Comment:
IIUC `Fatalf` is simply log to error and call `os.Exist(1)` - I think in this case we shouldn't stop the test suite altogether but simply log to error and continue. WDYT?
In any case this error should never be non-nil but its best to check.

Reply:
Yes, we can do that, I also don't expect this to fail.

Reply:
IMHO I think the tests should not fail if there's an error on the server-side (e.g. OCM returns HTTP 4xx/5xx). However, if we have an actual bug/error in our code, it might be a good idea to fail-fast. Especially since this is generating a request body which could skew the results of the test. Open for discussion though!